frontend/apps/marketing/Dockerfile

```diff
@@ -0,0 +1,82 @@
+# syntax=docker.io/docker/dockerfile:1
+# This file was built based off the Next.js recommended Dockerfile
+# See: https://github.com/vercel/next.js/tree/canary/examples/with-docker
+
+ARG NODE_VERSION=20 # LTS EOL: 2025-04-30
+ARG DEBIAN_VERSION=bookworm # LTS EOL: 2028-06-30
```

Comment:
I would prefer you used Ubuntu here. There's an advantage to having all/most of our dockerfiles use the same base distro. Its annoying but it means for example that we can copy and paste node installation commands when we syncrhonously upgrade (more "monorepo mind" stuff, annoying I know lol).

I too would default to Debian and prefer it for a base, but Ubuntu is similar enough I think its not worth having the variation in our codebase.

Reply:
I did swap the base OS to use Ubuntu (and install Node as a step). That being said, I do think there's value in using an official base image for Node which takes away the need to maintain security and random Node/OS integration issues. There is also value in having a series of Code.org base docker images in which we use a composition pattern, something like this? (cc @cat5inthecradle - we might have discussed this composition pattern in the past)

![image](https://github.com/user-attachments/assets/9b8fd35b-77cf-4e28-9536-2337887286c5)

Reply:
As much as I want to use `codedotorg/cdo-ci`, it does include additional packages and services that are not needed for the Next.js app (mysql, ruby, etc.). It does make sense to have Code.org global base OS though that uses the same base operating system